### ISSUE TITLE
Execise 6.1

### ISSUE THREAD

**Issue by RangerChu** [LyWangPX/Reinforcement-Learning-2nd-Edition-by-Sutton-Exercise-Solutions#79]
V_t denote the array of state values used at time t in the TD error (6.5) and in the TD update (6.2).
And delta_t is calculated at time t+1.

![QQ图片20210323111639](https://user-images.githubusercontent.com/16542762/112087746-7dc9da00-8bc9-11eb-867f-d5038cdf1380.jpg)

The agent only updates the V value of S_t at the time of t+1, and the V values of other states remain unchanged.

![QQ图片20210323111643](https://user-images.githubusercontent.com/16542762/112087753-815d6100-8bc9-11eb-970f-5d10a01dd0ff.jpg)

![1](https://user-images.githubusercontent.com/16542762/112087829-a8b42e00-8bc9-11eb-9919-a71a6bbe0254.jpg)

**Comment by ehddnr747** [comment]
You misunderstand the problem definition.
Even though delta_t is calculated at time step t+1 with R_[t+1] and S_[t+1], use call it V_t, which is used to calculate delta_t.

**Comment by zexiangliu** [comment]
I agree with @ehddnr747 that V_t is used to calculate delta_t instead of V_{t+1}. If that is fixed in @RangerChu 's answer, we should have a correct solution.
